code/modules/mob/living/simple_animal/friendly/drone/verbs.dm

```diff
@@ -12,21 +12,6 @@
 	to_chat(src, "<b>Drone Laws</b>")
 	to_chat(src, laws)
 
-/mob/living/simple_animal/drone/verb/toggle_light()
-	set category = "Drone"
-	set name = "Toggle drone light"
-	if(stat == DEAD)
-		to_chat(src, "<span class='warning'>There's no light in your life... by that I mean you're dead.</span>")
```

Comment:
poor pun, you served us well.

Reply:
I demand a new pun

Reply:
[rest well my sweet prince](https://www.youtube.com/watch?v=bq54jb_5WCI)